drivers/volume/mock/mock.go

```diff
@@ -102,6 +104,11 @@ func (m *Driver) GetStorageClassName() string {
 	return storageClassName
 }
 
+// GetClusterID returns the clusterID for the driver
+func (m *Driver) GetClusterID() (string, error) {
+	return clusterID, nil
```

Comment:
put clusterID in the Driver struct as the nodes are in there (i.e `m.clusterID`). This will allow us to create 2 mock clusters with different IDs.